cardea/modeling/modeler.py

```diff
@@ -311,97 +294,111 @@ def create_space(self, pipeline):
             space_list[primitive] = space
         return space_list
 
-    def get_block(self, pipeline):
-        blocks = pipeline.blocks
-        block = blocks.popitem()
-        self.block_instance = block[1]
-
     def hyperopt_train_test(self, params):
-        '''Creates the objective function to minimize.
+        """Creates the objective function to minimize.
 
         Args:
             params: The parameter for a specific pipleline.
 
         Returns:
             The the model secore after K-fold corss-validation.
-        '''
-        pipeline = self.create_pipeline(self.primitives, params)
-        self.get_block(pipeline)
-        model = self.block_instance.instance
-        if(self.problem_type == 'regression'):
-            return cross_val_score(
-                model, self.result['X_train'], self.result['y_train'], scoring='r2').mean()
-        return cross_val_score(model, self.result['X_train'], self.result['y_train']).mean()
+        """
+
+        pipeline = self.create_pipeline(self.primitive, params)
+
+        return self.kfold_scoring(self.data_frame, self.target, pipeline)
 
     def objective(self, params):
 
         accuracy = self.hyperopt_train_test(params)
-        return {'loss': -accuracy, 'status': STATUS_OK}
+        if not self.minimize_cost:
+            print("Maximum")
```

Comment:
I am not sure if this `print` was left on purpose or it was just for debugging while you were coding.

Reply:
It was for debugging, I'll remove it. Thanks!

Reply:
Solved in 08ace40fd3a2453a123e3361c18ec730ddbbd020